test/features/install_trustyui.feature

```diff
@@ -0,0 +1,16 @@
+@trustyui
+@infinispan
+@kafka
+Feature: Install Kogito Trusty UI
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
```

Comment:
```suggestion
    And Kogito Operator is deployed
```

Reply:
The Infinispan and Kafka aren't used in this case, no need to deploy their operators.